src/sdk.tsx

```diff
@@ -7,19 +7,30 @@ class SimpleSDK {
     origin: string = window.location.origin;
 
     constructor() {
-        this.listen(window);
+        // this.listen(window);
     }
 
     post(postEvent: postEvent, target: Window, origin: string = window.location.origin) {
         target.postMessage(postEvent, origin)
     }
 
-    listen(target: Window) {
+    listen(target: Window, senderUrl: string) {
```

Comment:
we should have this configurable, to avoid xss requests

Reply:
Sorry, how do you want to set this up?